src/utils/getTokenHoldersCount.ts

```diff
@@ -1,7 +1,16 @@
 export const getTokenHoldersCount = async (): Promise<number> => {
-  const res = await fetch(
-    'https://api.ethplorer.io/getTokenInfo/0x579CEa1889991f68aCc35Ff5c3dd0621fF29b0C9?apiKey=freekey',
-  )
-  const data = await res.json()
-  return data.holdersCount
+  try {
+    const res = await fetch(
+      'https://api.ethplorer.io/getTokenInfo/0x579CEa1889991f68aCc35Ff5c3dd0621fF29b0C9?apiKey=freekey',
+    )
+    const data = await res.json()
+    if (!data || typeof data.holdersCount !== 'number') {
```

Comment:
If there's no data case is better, don't test for the type.

Reply:
Any specific reason for that?

Reply:
Type might not come as a number, might be aa ``BigInt`` or ``string``, so it's not safe.